repo/index.yaml

```diff
@@ -93,6 +93,49 @@ entries:
     - azure-industrial-iot/azure-industrial-iot-0.1.0.tgz
     version: 0.1.0
   azuremonitor-containers:
+  - apiVersion: v1
+    appVersion: 7.0.0-1
+    created: "2020-11-12T13:55:43.6867268-08:00"
+    description: Helm chart for deploying Azure Monitor container monitoring agent
+      in Kubernetes
+    digest: a434da476574ac82ecc71c1f702981de119f9edaee53957fe277b7637d392b71
+    home: https://docs.microsoft.com/en-us/azure/monitoring/monitoring-container-health
+    icon: https://raw.githubusercontent.com/microsoft/Docker-Provider/ci_prod/img/azuremonitor-containers.svg
+    keywords:
+    - monitoring
+    - azuremonitor
+    - azure
+    - oms
+    - containerinsights
+    - metric
+    - event
+    - logs
+    - containerhealth
+    - kubernetesmonitoring
+    - acs-engine
+    - aks-engine
+    - azurestack
+    - openshift v4
+    - azure redhat openshift v4
+    - on-prem kubernetes monitoring
+    - containerlogs
+    - containerhealth
+    - containermonitoring
+    - hybrid kubernetes monitoring
+    - kubernetes
+    - kuberneteshealth
+    kubeVersion: ^1.10.0-0
+    maintainers:
+    - email: visnara@microsoft.com
+      name: vishiy
+    - email: gangams@microsoft.com
+      name: ganga1980
+    name: azuremonitor-containers
+    sources:
+    - https://github.com/microsoft/Docker-Provider/tree/ci_prod
+    urls:
+    - azuremonitor-containers/azuremonitor-containers-2.7.9.tgz
+    version: 2.7.9
   - apiVersion: v1
     appVersion: 7.0.0-1
     created: "2020-11-05T08:19:17.8990924-08:00"
```

Comment:
Please change the date to today's date. You can leave the time as is as it doesn't matter. Date shows up in artifact hub as well as RSS for each chart version.
![image](https://user-images.githubusercontent.com/10353076/99002689-b0641780-24f1-11eb-8e03-6994453eb170.png)

Reply:
> Please change the date to today's date. You can leave the time as is as it doesn't matter. Date shows up in artifact hub as well as RSS for each chart version.
> ![image](https://user-images.githubusercontent.com/10353076/99002689-b0641780-24f1-11eb-8e03-6994453eb170.png)

> your referring old previous chart release date. for 2.7.9 is  created: "2020-11-12T13:55:43.6867268-08:00"

Reply:
i see date created: "2020-11-05T08:19:17.8990924-08:00". I also see reduntant things - see lines 139,140,141
- apiVersion: v1
    appVersion: 7.0.0-1	    appVersion: 7.0.0-1
    created: "2020-11-05T08:19:17.8990924-08:00"	    created: "2020-11-05T08:19:17.8990924-08:00"

Reply:
> i see date created: "2020-11-05T08:19:17.8990924-08:00". I also see reduntant things - see lines 139,140,141
> 
> * apiVersion: v1
>   appVersion: 7.0.0-1
>   created: "2020-11-05T08:19:17.8990924-08:00"

>  line 139,140 and 141 for 2.7.8 release. If you see the full file, you see the created date  top for 2.7.9 on the top at line 98.

2.7.9 
https://github.com/microsoft/charts/blob/842825203202fd82fed3188fc23b536f983b96d6/repo/index.yaml#L98

2.7.8
https://github.com/microsoft/charts/blob/842825203202fd82fed3188fc23b536f983b96d6/repo/index.yaml#L141